content/providers/provider-azure/authentication.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Authentication 
+weight: 10
+description: Authentication options with the Upbound Azure official provider
+---
+
+The Upbound Official Azure Provider supports multiple authentication methods.
+
+* [Service principal with Kubernetes secret]()
+* [System-assigned managed identity]()
+* [User-assigned managed identity]()
+
+## Service principal with Kubernetes secret
+
+A service principal is an application within the Azure Active Directory that
+passes `client_id`, `client_secret`, and `tenant_id` authentication
+tokens to create and manage Azure resources. 
+
+### Create a service principal using the Azure CLI tool
+
+First, find the Subscription ID for your Azure account.
+
+```shell
+az account list
+```
+
+Note the value of the `id` in the return output. 
+
+Next, create a service principle `Owner` role. Update the `<subscription_id>`
+with the `id` from the previous command.
+
+```shell
+az ad sp create-for-rbac --sdk-auth --role Owner --scopes /subscriptions/<subscription_id> \
+  > azure.json
+```
+
+The `azure.json` file in the preceding command contains the client ID, secret, and
+tenant ID of your subscription.
+
+
+Next, use `kubectl` to associate your Azure credentials file with a generic
+Kubernetes secret.
+
+```shell
+kubectl create secret generic azure-secret -n upbound-system --from-file=creds=./azure.json
+```
+
+## Configure your provider
+
+Apply these changes to your `ProviderConfig` file. 
+
+```yml
+apiVersion: azure.upbound.io/v1beta1
+metadata:
+  name: default
+kind: ProviderConfig
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-secret
+      key: creds
+```
+
+Your credential `source` must be `Secret` and you must specify the namespace,
```

Comment:
```suggestion
```yml {label="secretPC"}
apiVersion: azure.upbound.io/v1beta1
metadata:
  name: default
kind: ProviderConfig
spec:
  credentials:
    source: Secret
    secretRef:
      namespace: upbound-system
      name: azure-secret
      key: creds
```

Your credential {{<hover label="secretPC" line="7">}}source{{</hover>}} must be {{<hover label="secretPC" line="7">}}Secret{{</hover>}} and you must specify the {{<hover label="secretPC" line="9">}}namespace{{</hover>}},
```

Optionally add a label to the code block (`{label="blahblah"}`) and use the `hover` shortcode to enable hover to highlight between the inline command and the code block.

I would call out not just the namespace but that this is the name of the Kubernetes secret, the key name inside the object and the namespace (if it's not default). [Example](https://docs.upbound.io/providers/provider-aws/authentication/#create-a-providerconfig)